probe-rs/targets/VA108xx_Series.yaml

```diff
@@ -0,0 +1,167 @@
+name: VA108xx Series
+generated_from_pack: true
+pack_file_release: 1.4.0
+variants:
+- name: VA108xx
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Ram
+    name: DRAM
+    range:
+      start: 0x10000000
+      end: 0x10008000
+    cores:
+    - main
+  - !Nvm
+    name: NVM
+    range:
+      start: 0x0
+      end: 0x20000
+    is_boot_memory: true
+    cores:
+    - main
+  flash_algorithms:
+  - va108xx_fm25v20a_fram_128kb_prog
+  - va108xx_m95m01_128kb_prog
+  - va108xx_mr25h10_1mb_prog
+  - va108xx_ttflash_prog
+- name: VA108xx_RAM
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Ram
+    name: DRAM
+    range:
+      start: 0x10000000
+      end: 0x10008000
+    cores:
+    - main
+  - !Ram
+    name: IRAM
+    range:
+      start: 0x0
+      end: 0x20000
+    is_boot_memory: true
```

Comment:
You'll need to use the git version of target-gen, I'm afraid.

Reply:
Ah yes, I generated this with an old version..

Reply:
Fixed. Is the assumption to remove `write: false` for the IRAM correct? (to allow RAM boot)

Reply:
Yes, IRAM needs to be writeable for your use case.